queries/buildings.jinja2

```diff
@@ -1,12 +1,9 @@
 SELECT
     name,
     way_area::bigint AS area,
+    COALESCE("building", "amenity", "shop", "tourism") AS kind,
```

Comment:
I believe in some cases (I thought including buildings, could also be a
regression?), we were only including non-"yes" values. Because it's not
useful as Matt said, it takes up space (a lot on low zooms!), and it's
ambiguous when coalescing with other tags with possible "yes" values.


On Friday, September 25, 2015, Matt Amos <notifications@github.com> wrote:

> In queries/buildings.jinja2
> <https://github.com/mapzen/vector-datasource/pull/246#discussion_r40419590>
> :
>
> > @@ -1,12 +1,9 @@
> >  SELECT
> >      name,
> >      way_area::bigint AS area,
> > +    COALESCE("building", "amenity", "shop", "tourism") AS kind,
>
> Over 80% of buildings
> <http://taginfo.openstreetmap.org/keys/building#values> are tagged
> building=yes, which will mean that over 80% of building polygons in the
> tiles will have a kind of "yes", which is probably okay because they're in
> a separate layer, but doesn't seem to me like it's a useful value. Would it
> be better to keep the "building", "amenity", "shop" and "tourism" columns
> in the query and normalise them in a Python filter? That way we could
> express slightly more complex and useful logic about collapsing these
> values.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mapzen/vector-datasource/pull/246/files#r40419590>.
>